### PR TITLE
Fix #491 Implemented event import and export

### LINKED ISSUE #491: Ability to export and import events by CSV files
The feature would help with courses that have a large number of events (eg. some UBC CHBE courses have 40) that would need to be rolled over into a new course with some minor changes (eg. dates, descriptions, groups assigned).

Editing the events locally with a program (as opposed to an in-browser rollover feature) would be preferred because it allows for easy table navigation in a spreadsheet program (eg. batch copy and pasting event descriptions, close dates, etc).

### PR DESCRIPTION
**Pull request by gondek**
Includes model and Selenium tests. Functionality is available through a course's menu. Access table patch included.

### CODE REVIEW

File: app/config/sql/delta_x.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+--
+-- add permissions for /event/export and /event/import
+--
+
+INSERT INTO acos (id, parent_id, model, foreign_key, alias, lft, rght) VALUES
```
Comment (xcompass) [inline app/config/sql/delta_x.sql]: Could you change the file name to delta_10.sql?

**Comment by gondek** [comment]
Renamed!

**Comment by gondek** [comment]
@xcompass Can we fix this: https://github.com/ubc/iPeer/commit/fbfce490f6deb03c9ee1d96bea8236e3278de405#commitcomment-10323950

**Comment by xcompass** [comment]
Good cache. I'll fix it.

Cheers,
Pan

**Pull request merged**